week03/html/project.html

```diff
@@ -0,0 +1,71 @@
+<link href="../css/project.css" rel="stylesheet" /> 
+<link rel="icon" type="image/png" sizes="16x16" href="../src/billyLogo.png"/>
+
+<title>Lee Gabin</title>
+
+<body>
+    <!--상단바-->
+    <header id="header">
+        <div class="navbar">
+        <div class="nav1">
+        <a href="../html/home.html" class="logo">
+        <img src="../src/mainLogo.png" height="70px" />
+    </a>
+    </div>
+    <div class="nav2"></div>
+    <div class="nav3">
+    <ul>
+        <li><a href="../html/home.html">Home</a></li>
+        <li><a href="../html/about.html">About</a></li>
+        <li><a href="../html/project.html" style="color:black">Project</a></li>
+        <li><a href="../html/activity.html">Activity</a></li>
+        <li><a href="../html/contact.html">Contact</a></li>
+    </ul>
+    </div>
+    </div>
+    </header>
+
+    <!--내용-->
+    <div class="MyProjects">
+        <div class="title">Project</div>
+        <div class="projectType">
+            <ul>
+                <li class="game"><a href="../html/project.html" style="color: #0D51FE;">Game</a></li>
+                <li><a href="../html/project_web.html">Web</a></li>
+            </ul>
+            <div class="gamelist"> 
+            <div class="projectCard1">
+                <img src="../src/sunthelight.png" height="180px">

```

Comment:
태그를 잘 닫았는지 확인하면 좋겠습니다.

```suggestion
                <img src="../src/sunthelight.png" height="180px" />
```